BlueShift-iOS-SDK/BlueShift.m

```diff
@@ -899,6 +899,18 @@ - (BOOL)isTrackingEnabled {
     return [status isEqual:kYES] ? YES : NO;
 }
 
+#pragma mark Opt out push notifications
+- (void)optOutFromPushNotifications:(BOOL)isOptedOut {
+    [BlueShiftAppData currentAppData].enablePush = isOptedOut;
```

Comment:
I think this is incorrect. When you say you want to opt out, you will be passing YES/true here. If you set that to enablePush, it is basically an opt-in.

Reply:
Thanks for bringing it to notice, I will modify it based on the finalized method names.